src/main/java/org/folio/qm/converter/field/dto/BibliographicDataFieldConverter.java

```diff
@@ -13,8 +15,11 @@
 import org.springframework.stereotype.Component;
 
 @Component
+@RequiredArgsConstructor
 public class BibliographicDataFieldConverter extends CommonDataFieldConverter {
   private static final char AUTHORITY_ID_SUBFIELD_CODE = '9';
+  private static final char AUTHORITY_NATURAL_ID_SUBFIELD_CODE = '0';
+  private final LinksClient linksClient;
```

Comment:
So, this is not used in a class and can be removed. As well as @RequiredArgsConstructor